docs/SPIR-V.rst

```diff
@@ -1086,6 +1086,12 @@ The following intrinsic HLSL functions are currently supported:
 - ``reversebits``: Reverses the order of the bits, per component. Uses SPIR-V ``OpBitReverse``.
 - ``clip``: Discards the current pixel if the specified value is less than zero. Uses conditional
   control flow as well as SPIR-V ``OpKill``.
+- ``ddx``: Partial derivative with respect to the screen-space x-coordinate. Uses SIR-V ``OpDPdx``.
```

Comment:
I feel that we can condense some of the items translating into SPIR-V opcode into a table. No need to do it right now, though.